crates/telio-wg/src/wg.rs

```diff
@@ -115,6 +122,86 @@ pub struct Io {
     pub libtelio_wide_event_publisher: Option<mc_chan::Tx<Box<LibtelioEvent>>>,
 }
 
+/// Keepalive packets size
+pub const KEEPALIVE_PACKET_SIZE: u64 = 32;
+/// Default wireguard keepalive duration
+pub const WG_KEEPALIVE: Duration = Duration::from_secs(10);
+
+#[derive(Copy, Clone)]
+/// docs
+pub struct BytesAndTimestamps {
+    /// docs
+    rx_bytes: u64,
+    /// docs
+    tx_bytes: u64,
+    /// docs
+    rx_ts: Option<Instant>,
+    /// docs
+    tx_ts: Option<Instant>,
+}
+
+impl BytesAndTimestamps {
+    /// docs
+    pub fn new(rx_bytes: Option<u64>, tx_bytes: Option<u64>) -> Self {
+        let now = Instant::now();
+        BytesAndTimestamps {
+            rx_bytes: rx_bytes.unwrap_or_default(),
+            tx_bytes: tx_bytes.unwrap_or_default(),
+            rx_ts: if rx_bytes.is_none() { None } else { Some(now) },
```

Comment:
I believe, we should check both, `rx_bytes.is_none()` as well as `rx_bytes == 0`. And of course, same for `tx`.

Reply:
fixed